.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,18 @@
+name: Shellcheck
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Run ShellCheck
+      uses: ludeeus/action-shellcheck@2.0.0
+      env:
+        SHELLCHECK_OPTS: -x # allow outside sources
+      with:
+        ignore_paths: >-
+          secp256k1-sys
```

Comment:
I'm acking but I don't know this syntax `>-`  - is that just how one makes a newline in yml? I thought it was just `>`

Reply:
Yes also new for me. It was mentioned in the [README of the GH action](https://github.com/ludeeus/action-shellcheck/blob/00b27aa7cb85167568cb48a3838b75f4265f2bca/README.md?plain=1#L47-L57):

> You can use the `ignore_paths` and `ignore_names` input to disable specific 
> directories and files.
> These are passed as environment variables, and should evaluate to a single space-separated string.
> It may be convenient to use [`>-`](https://yaml.org/spec/1.2.2/#65-line-folding) for readability if you have multiple selectors.
> 
> 
> ```text
> sample structure:
> sample/directory/with/files/ignoreme/test.sh
> sample/directory/with/files/ignoremetoo/test.sh
> sample/directory/with/files/test.sh
> sample/directory/with/files/ignorable.sh
> ```

This is the [YAML Line folding](https://yaml.org/spec/1.2.2/#65-line-folding)

Reply:
Yeah let's try to not get smart with YAML. We'll shoot ourselves in the foot. I am not using the `>-` anymore.